src/main/java/org/lobid/resources/MabXmlHandler.java

```diff
@@ -0,0 +1,81 @@
+/** Copyright 2013,2014 hbz
+ * Licensed under the Eclipse Public License 1.0 
+ **/
+
+package org.lobid.resources;
+
+import org.culturegraph.mf.framework.DefaultXmlPipe;
+import org.culturegraph.mf.framework.StreamReceiver;
+import org.culturegraph.mf.framework.XmlReceiver;
+import org.culturegraph.mf.framework.annotations.Description;
+import org.culturegraph.mf.framework.annotations.In;
+import org.culturegraph.mf.framework.annotations.Out;
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+
+/**
+ * A MAB XML reader.
+ * 
+ * @author Pascal Christoph (dr0i)
+ * 
+ */
+@Description("A MAB XML reader")
+@In(XmlReceiver.class)
+@Out(StreamReceiver.class)
+public final class MabXmlHandler extends DefaultXmlPipe<StreamReceiver> {
```

Comment:
We have `AlephMabXmlHandler` in metafacture-core:
https://github.com/hbz/metafacture-core/blob/master/src/main/java/org/culturegraph/mf/stream/converter/xml/AlephMabXmlHandler.java